tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -742,12 +742,15 @@ private void createWildFlyScript() throws Exception {
                 secretKeyAlias = "key";
             }
             String createScriptCheck = "";
-            if (Paths.get(String.format("%s.cli", fileSystemRealmName)).toFile().exists()) {
+
+            Path scriptPath = Paths.get(String.format("%s/%s.cli", outputRealmLocation, fileSystemRealmName));
+
+            if (scriptPath.toFile().exists()) {
                 createScriptCheck = prompt(
                         false,
                         null,
                         false,
```

Comment:
```suggestion
                createScriptCheck = prompt(
                        true,
                        ElytronToolMessages.msg.shouldFileBeOverwritten(scriptPath.toString()),
                        false,
```

The prompt currently reads `Password: `, and hides the user's input. These changes should actually display the message and make input visible ([see the Javadoc for the `prompt` method](https://github.com/wildfly-security/wildfly-elytron/blob/a335699df3b55fd5f9258fe480670cbce8b7394a/tool/src/main/java/org/wildfly/security/tool/Command.java#L100-L110)).

We probably missed this when the command was originally added, but if you could fix it here that would be great.